util/slugToTitleCase.ts

```diff
@@ -0,0 +1,3 @@
+export default function slugToTitleCase(slug: string): string {
```

Comment:
Worth noting... 

1. We don't have consistent slugs right now. For example, there is `literal-search-patterns` and `how-to-add-open-source-software-projects-to-sourcegraph`. We have an [Asana ticket](https://app.asana.com/0/1200198213175654/1200550968789812/f) to evaluate URLs.
2. There is Issue #112 which is open for discussion about sentence case vs. title case, which we should discuss more with Content and other teams.

Reply:
Thanks for these links! I debated actually calling this method `authorToTitleCase` or some such, since right now it's only applied to author names. I thought it could be more widely useful, so opted for this, but let me know if you think I should change the name to make it clear that we haven't decided on that issue yet.